### PR TITLE
ToDo List Milestone 1: Adding Static Tasks using the JS Object

### PR DESCRIPTION
**Pull request by AyoubM490**
In this milestone, I:
- Used Webpack for bundling
- Used ES6

### CODE REVIEW

File: src/modules/addTask.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    } else {
+      label.classList.remove('checked');
+      task.bool = 'false';
+    }
+  };
+
+  input.addEventListener('click', inputChecked);
+};
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (brytebee) [inline src/modules/addTask.js]: - [x] **Required:** It is not a good practice to disable the linters. The linters make a team code uniform. Kindly fix it.

**Hint:** Try using `export default addTask`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    description: 'Complete To Do list project',
+    bool: false,
+    index: 2,
+  },
+];
+
+const addTaskStatic = (tasks) => {
+  tasks.forEach((task) => {
+    addTask(task);
```
Comment (brytebee) [inline src/index.js]: - [x] **Required:** Task should be displayed in the order of their index.

image

**Hint:** Use the sort function. For example `tasks.sort((a, b) => a.index - b.index)`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @AyoubMs & @ayubch 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- webpack correctly configured✔️
- no linter errors✔️
- no console errors✔️
- correctly using the HTML Webpack Plugin✔️
- modular codebase✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

- [ ] **Required:** You have an empty README file. Kindly document your project using the [Microverse Template.](https://github.com/microverseinc/readme-template/blob/master/README.md)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addTask.js (hunk 1 above)
Comment (AyoubM490) [reply src/modules/addTask.js]: Done

File: src/index.js (hunk 2 above)
Comment (AyoubM490) [reply src/index.js]: Done

**Review by dasileker — APPROVED** [review]
## **Hi @AyoubMs,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**